java-client/src/test/java/org/opensearch/client/opensearch/integTest/ClusterClientIT.java

```diff
@@ -127,7 +127,7 @@ public void testClusterGetSettings() throws IOException {
 
         GetClusterSettingsResponse getSettingsResponse = openSearchClient.cluster()
                 .getSettings(new GetClusterSettingsRequest.Builder().build());
-        assertEquals(1, getSettingsResponse.persistent().size());
+        assertTrue(getSettingsResponse.persistent().containsKey("cluster"));
```

Comment:
What does size return here? 
Does it return the number of keys present?  (I see the return type is `GetClusterSettingsResponse`)

Reply:
Yes it returns the size of the map

Reply:
I agree checking on the cluster response.
Just curious, is the flakiness caused by the plugin sets the state sometimes and it doesnt at other times?

Reply:
Yeah thats what I saw by running the test multiple times, the setting is present just some of the time.